collections/_programs/hacktoberfest.html

```diff
@@ -8,7 +8,7 @@
 permalink: /programs/hacktoberfest
 excerpt: "Layer5 - Hacktoberfest"
 ---
-<img src="/assets/images/programs/hacktoberfest2020/hacktober.svg" class="image-center" />
+<img src="https://user-images.githubusercontent.com/7570704/94851172-b1336500-03ed-11eb-917c-9bdb67ba8684.png" alt="Hacktoberfest and Layer5 logo" class="image-center" />
```

Comment:
Can you download the graphic in assets folder and then add it? Also the height to width ratio in current preview looks disturbed.

Reply:
No problem, that makes more sense!

Reply:
The width for the image is set at 40%, I set it to 60% in chrome devtools and it does seem to look better. Would you say that's the best way of improving the issue that you mentioned ?

Reply:
I think @leecalcote and @chandrashritii can look into if we could have a wider graphic.